jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java

```diff
@@ -3,25 +3,151 @@
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.buttonFrame;
-import static io.github.com.StaticSite.inputButtonDefaultPage;
-import static io.github.com.StaticSite.inputButtonDisabledPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.inputs.ButtonFrame.*;
 
 public class ButtonTests extends TestsInit {
 
     @Test
     public void defaultButtonTest() {
-        inputButtonDefaultPage.open();
+        openSection("Buttons");
 
-        buttonFrame.button.is().enabled();
-        buttonFrame.button.click();
-        buttonFrame.button.has().text("BUTTON");
+        button.getContainedButtonByIndex(1).click();
+        button.getContainedButtonByIndex(1).is().text("DEFAULT");
+        checkContainedField.is().text("Last click: Default");
+
+        button.getContainedButtonByIndex(2).click();
+        button.getContainedButtonByIndex(2).is().text("PRIMARY");
+        checkContainedField.is().text("Last click: Primary");
+
+        button.getContainedButtonByIndex(3).click();
+        button.getContainedButtonByIndex(3).is().text("SECONDARY");
+        checkContainedField.is().text("Last click: Secondary");
+
+        button.getContainedButtonByIndex(4).is().text("DISABLED");
+        button.getContainedButtonByIndex(4).is().attr("disabled");
+        button.getContainedButtonByIndex(4).is().disabled();
+
+        button.getContainedButtonByIndex(5).click();
+        button.getContainedButtonByIndex(5).is().text("LINK");
+        checkContainedField.is().text("Last click: Link");
+        button.getContainedButtonByIndex(5).is().notVisible();
```

Comment:
Please check that it's contained buttons(for all buttons) - is().hasClass("MuiButton-contained"). The same for other tests